ref.yml

```diff
@@ -107,6 +107,18 @@ fields:
         - README.md
         - index.md
         - coq.opam
+  - chat:
+      description: >
+        Url to the chat room associated with this project.
+      required: false
+      used:
+        - README.md
+      item_fields:
+        - shield:
+            description: >
+              Badge associated with the website of the chat room.
+            default: ""
+            examples: zulip, matrix, discourse
```

Comment:
The `url` field item is missing. Besides, I would change the description to something more precise:
- for the `chat` variable: adds a badge with a link to a chat room
- for the `shield` variable: name of the chat to display in the shield icon.

Reply:
> The url field item is missing.

Oops. I wonder why it worked ^_^. Sorry about that, the semantics is not yet clear to me.

Reply:
Well, `ref.yml` is not actually checked against anything, so...